src/engraving/layout/layoutsystem.cpp

```diff
@@ -391,18 +391,20 @@ System* LayoutSystem::collectSystem(const LayoutOptions& options, LayoutContext&
     qreal stretchCoeff = 1;
     qreal prevWidth = 0;
     int iter = 0;
-    static double epsilon = score->spatium() * 0.08; // For reference: this is approximately as small as the width of a note stem
-    static constexpr int maxIter = 200; // Limits the number of iterations, just for safety. In reality, most systems require less then 10 iterations.
-    static constexpr float multiplier = 1.5; // Empirically optimized value which allows the fastest convergence of the following algorithm.
+    static double epsilon = score->spatium() * 0.05; // For reference: this is smaller than the width of a note stem
```

Comment:
This shouldn't be static because `score` is not static either. (otherwise, the spatium of the very first score in the program will be used for the rest of all time)

Reply:
Thanks, that dates back to my first PR. I'll fix that